src/components/container/RelativeActivity/RelativeActivity.tsx

```diff
@@ -0,0 +1,184 @@
+import { faChevronDown, faChevronUp } from "@fortawesome/free-solid-svg-icons";
+import { add, startOfDay } from "date-fns";
+import React, { useContext, useEffect } from "react";
+import { useState } from "react";
+import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon";
+import getDayKey from "../../../helpers/get-day-key";
+import { DailyDataQueryResult, queryDailyData } from "../../../helpers/query-daily-data";
+import { ActivityMeter, CardTitle, DateRangeContext, } from "../../presentational";
+import "./RelativeActivity.css"
+import language from "../../../helpers/language";
+import { useInitializeView } from "../../../helpers/Initialization";
+import { ColorDefinition } from "../../../helpers/colors";
+
+export interface RelativeActivityProps {
+    dataTypes: RelativeActivityDataType[];
+    previewState?: "Default";
+    title?: string;
+    innerRef?: React.Ref<HTMLDivElement>
+    specifyThresholds?: boolean;
```

Comment:
This property seems to act more like `requireThresholds`.  The data type thresholds can be specified regardless of this setting, but when this is `true`, those without a threshold are hidden.

Reply:
I refactored this property away so it just does stuff automatically based on whether thresholds are set or not